src/AutoMapper/Execution/DelegateBasedResolver.cs

```diff
@@ -4,30 +4,29 @@ namespace AutoMapper.Execution
 {
     using System;
 
-    public class DelegateBasedResolver<TSource, TMember> : IMemberResolver
+    public class DelegateBasedResolver<TSource, TMember> : IValueResolver
```

Comment:
DelegateBasedResolver uses ResolutionContext, and for simple mapping using expression translation, I'm only using functions of TSource and TMember to get and set values.  Maybe in the future it can but for now I'm ignoring translating anything with ResolutionContext.